HHS/Head-Start-TTADP
# To prod > Grantee search fixes, don't close filter menu on blur

## Description of change
 
Recipient search results page now properly sorts the complete list of recipients.

 * Single SQL query hits the DB
 * Grouping by `regionId` allows recipients to be split on region
 * Sorting by region now has a secondary sort on name. Regional office defaults to sorting by region
 * Program/Grant specialists are ordered and put in a comma separated string in SQL
 * Updated frontend to just display result from the API

Also - the filters panel now won't close on blur (clicking on something besides the filter menu). The only ways to close it should be to successfully apply or cancel.

## How to test

View the grantee search page in staging. You should be able to search/sort by region/recipient name from the complete list of recipients, not just those listed when the page loads.

Go to the Recipient record page, open the filter menu. Click around not on the menu, observe with satisfaction that the menu does not close.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-478
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-499

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
